fdbcli/ConfigureCommand.actor.cpp

```diff
@@ -173,6 +173,16 @@ ACTOR Future<bool> configureCommandActor(Reference<IDatabase> db,
 		fprintf(stderr, "ERROR: These changes would make the configuration invalid\n");
 		ret = false;
 		break;
+	case ConfigurationResult::STORAGE_MIGRATION_DISABLED:
+		fprintf(stderr,
+		        "ERROR: Storage engine type cannot be changed because "
+		        "storage_migration_mode=disabled.\n");
+		fprintf(stderr,
+		        "Type `configure perpetual_storage_wiggle=1 storage_migration_type=gradual' to enable gradual "
+		        "migration with the perpetual wiggle, or `configure "
+		        "storage_migration_type=aggressive' for aggressive migration.\n");
+		ret = true;
```

Comment:
I reversed this flag after refactoring,
now return `false` means failed.
`ret = false;`